add-serve-target.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e -x
+
+PARAMETER_VALIDATION_PROMPT="
+Please supply the following arguments:
+
+  -w <path to workspace>: absolute path to the workspace with which to test the plugin
+  -a <app name>: the name of the app with which to test the plugin
+  
+For example: 
+  ./add-serve-target.sh -w path/to/workspace -a example-app"
+
+while getopts w:a: flag
+do
+    case "${flag}" in
+        w) workspace=${OPTARG};;
+        a) app=${OPTARG};;
+    esac
+done
+
+if [ -z $workspace ] || [ -z $app ]; then
+    echo "$PARAMETER_VALIDATION_PROMPT"
+    exit 1
+fi
```

Comment:
Better to keep indentation consistent: 2 spacess
```suggestion
do
  case "${flag}" in
    w) workspace=${OPTARG};;
    a) app=${OPTARG};;
  esac
done

if [ -z $workspace ] || [ -z $app ]; then
  echo "$PARAMETER_VALIDATION_PROMPT"
  exit 1
fi
```

Reply:
Updated!